src/demo-app/a11y/checkbox/checkbox-a11y.html

```diff
@@ -1 +1,42 @@
-Checkbox Placeholder
\ No newline at end of file
+<section>
+  <h2>Checkbox without label</h2>
+  <md-checkbox aria-label="A single checkbox with no label example"></md-checkbox>
+</section>
+<section>
+  <h2>Standalone checkbox</h2>
+  <md-checkbox>A simple checkbox</md-checkbox>
+  <md-checkbox [disabled]="true">Disabled checkbox</md-checkbox>
+  <md-checkbox [indeterminate]="true">An indeterminate checkbox</md-checkbox>
+</section>
+
+<section>
+  <h2>Pseudo checkboxe</h2>
+  <p>These pseudo checkboxes are for display only</p>
+  <md-pseudo-checkbox></md-pseudo-checkbox>
```

Comment:
Psuedo-checkbox is designed specifically to _not_ be an accessible checkbox so it shouldn't be used in the demo. It's mainly meant to be used inside of `role="option"` elements that express their selection via `aria-selected`